src/wasm-delegations-fields.h

```diff
@@ -93,7 +93,7 @@
 #ifndef DELEGATE_FIELD_CHILD_VECTOR
 #ifdef DELEGATE_GET_FIELD
 #define DELEGATE_FIELD_CHILD_VECTOR(id, name)                                  \
-  for (Index i = 0; i < (DELEGATE_GET_FIELD(id, name)).size(); i++) {          \
+  for (int i = int((DELEGATE_GET_FIELD(id, name)).size()) - 1; i >= 0; i--) {  \
```

Comment:
Are we confident that iterating through the vector in reverse is always going to be the expected default behavior? I guess this is consistent with the order in which other children are visited, so it shouldn't be a problem.

Reply:
Yes, if we need the opposite order we may want to add another option. I can't think of a use case atm for that though.